src/pages/RFOX/components/AddressSelection.tsx

```diff
@@ -184,13 +185,18 @@ export const AddressSelection: FC<AddressSelectionProps> = ({
     if (isManualAddress) return null
 
     return (
-      <AccountDropdown
-        defaultAccountId={maybeDefaultRuneAccountId}
-        assetId={thorchainAssetId}
-        onChange={handleAccountIdChange}
-        boxProps={boxProps}
-        buttonProps={buttonProps}
-      />
+      <InlineCopyButton
+        isDisabled={!maybeDefaultRuneAccountId}
+        value={maybeDefaultRuneAccountId ? fromAccountId(maybeDefaultRuneAccountId).account : ''}
```

Comment:
blocking: This will always copy the *default* rune AccountId, this should match the currently selected AccountId (i.e the one set in `handleAccountIdChange` in `<AddressSelection />` and `onRuneAddressChange` in `<StakeInput/>`).

solution: Pass down currently selected RUNE addy

Reply:
Passed done this selected address in [f36735f](https://github.com/shapeshift/web/pull/7689/commits/f36735fd3a0e273f0c87a554d23d6b78326a8d6c)